### PR TITLE
Issue13 and Issue15

### PR DESCRIPTION
**Pull request by daniloaf**

### CODE REVIEW

**Comment by amaurymedeiros** [comment]
PEP 0008 says to use no more than 79 characters. https://www.python.org/dev/peps/pep-0008/#id14

**Comment by daniloaf** [comment]
I know that. I just prefer 120 for a better visualization. There are other
lines with more than 80 characters.

But it's better using pep8 recommendations.
Em 31/12/2015 12:45, "Amaury" notifications@github.com escreveu:

File: moviesnetwork/moviesdata/data.py [hunk 1]
```diff
@@ -1,14 +1,56 @@
...
-            actor.movies = imdb_person.data['actor']
+            for movie in imdb_person.data.get('actor', []):
+                if movie.data['kind'] == 'movie':
+                    actor.movies.append(Movie.create(movie, False))
+            actor.headshot = imdb_person.get('headshot')
 
         return actor
+
+    def __eq__(self, other):
+        return isinstance(other, self.__class__) and self.actor_id == other.actor_id and self.name == other.name
```
Comment (amaurymedeiros) [inline moviesnetwork/moviesdata/data.py]: 80+

File: moviesnetwork/moviesdata/tests/test_services.py [hunk 2]
```diff
@@ -0,0 +1,71 @@
...
+from moviesdata.data import Actor
+from moviesdata.services import search_actor, get_actor_info
+
+
+class ServicesTests(TestCase):
+
+    @classmethod
+    def setUpClass(cls):
+        super(ServicesTests, cls).setUpClass()
+        cls.imdb_search_person_patcher = mock.patch('imdb.parser.http.IMDbHTTPAccessSystem.search_person')
```
Comment (amaurymedeiros) [inline moviesnetwork/moviesdata/tests/test_services.py]: 80+

File: moviesnetwork/moviesdata/tests/test_serializers.py [hunk 3]
```diff
@@ -0,0 +1,61 @@
...
+
+
+class MovieSerializerTests(TestCase):
+
+    def test__serializer__full_movie__serialized_data(self):
+        # Arrange
+        movie = Movie.create(IMDbMovie(
+            movieID=1,
+            data={'title': 'mymovie', 'year': 2015, 'kind': 'movie', 'rating': 10, 'cover url': 'cover'}), True)
+        expected_movie = {'movie_id': 1, 'title': 'mymovie', 'year': 2015, 'rating': 10, 'cover': 'cover'}
```
Comment (amaurymedeiros) [inline moviesnetwork/moviesdata/tests/test_serializers.py]: Both with 80+

**Comment by amaurymedeiros** [comment]
I pointed out few other lines > 80. The problem is it makes difficult to maintain and review. I have a 15" retina display and the line is wrapped when I try to review in github. Also, shorter lines help to split the screen in half, which most developers do.

File: moviesnetwork/moviesdata/data.py [hunk 4]
```diff
@@ -1,14 +1,56 @@
...
+        """
+        Creates a new actor using an imdb.Person.Person instance.
+
+        full_info: False for id and name only. True for more info.
+        """
         actor = Actor(imdb_person.getID(), imdb_person.data['name'])
         if full_info:
-            actor.movies = imdb_person.data['actor']
+            for movie in imdb_person.data.get('actor', []):
+                if movie.data['kind'] == 'movie':
```
Comment (amaurymedeiros) [inline moviesnetwork/moviesdata/data.py]: List comprehension here

File: moviesnetwork/moviesdata/data.py [hunk 5]
```diff
@@ -1,14 +1,56 @@
...
+        self.movie_id = movie_id
+        self.title = title
+        self.year = year
+        self.rating = None
+        self.cover = None
+
+    @staticmethod
+    def create(imdb_movie, full_info):
+        """
+        Creates a new actor using an imdb.Person.Person instance.
```
Comment (amaurymedeiros) [inline moviesnetwork/moviesdata/data.py]: Review this comment

File: moviesnetwork/moviesdata/views.py [hunk 6]
```diff
@@ -16,3 +16,17 @@ def get(self, request):
...
+
+    def get(self, request, actor_id):
+        """
+        Gets an actor full information.
+        """
+        actor = get_actor_info(actor_id)
+        if actor:
+            serializer = ActorSerializer(actor)
+            return response.Response(serializer.data)
+        else:
```
Comment (amaurymedeiros) [inline moviesnetwork/moviesdata/views.py]: you don't need the else

**Pull request merged**